src/BibleWell.Aquifer.Api/Client/Models/Request2.cs

```diff
@@ -0,0 +1,85 @@
+// <auto-generated/>
+#pragma warning disable CS0618
+using Microsoft.Kiota.Abstractions.Extensions;
+using Microsoft.Kiota.Abstractions.Serialization;
+using System.Collections.Generic;
+using System.IO;
+using System;
+namespace BibleWell.Aquifer.API.Client.Models
+{
+    [global::System.CodeDom.Compiler.GeneratedCode("Kiota", "1.0.0")]
+    #pragma warning disable CS1591
+    public partial class Request2 : IParsable
```

Comment:
The client generation process as documented doesn't clean up files before generating, it just overwrites any existing files.  This class needs to be deleted because it was replaced by `CreateDeviceInstallationRequest` but never cleaned up.  If you want you can delete the `Client` folder's contents and regen.